internal/config/client.go

```diff
@@ -28,9 +29,10 @@ var (
 
 // MongoDBClient contains the mongodbatlas clients and configurations
 type MongoDBClient struct {
-	Atlas   *matlasClient.Client
-	AtlasV2 *atlasSDK.APIClient
-	Config  *Config
+	Atlas      *matlasClient.Client
+	AtlasV2    *atlasSDK.APIClient
+	OldAtlasV2 *oldAtlasSDK.APIClient
```

Comment:
I think it'd be useful to add some comment about what this is and why we need it

Reply:
I added a comment to make it more clear